taqueria-vscode-extension/src/test/suite/extension.test.ts

```diff
@@ -1,107 +1,143 @@
 import assert from 'assert';
+import { execSync } from 'child_process';
```

Comment:
This doesn't look like it is getting used in this file

Reply:
Fixed.